tests/integration/relation_tests/new_relations/test_charm_relations.py

```diff
@@ -99,6 +99,16 @@ async def test_database_relation_with_charm_libraries(ops_test: OpsTest):
     )
     await ops_test.model.wait_for_idle(apps=APP_NAMES, status="active")
 
+    await ops_test.model.block_until(
+        lambda: is_relation_joined(
+            ops_test,
+            FIRST_DATABASE_RELATION_NAME,
+            "database",
```

Comment:
nit: can you please add a constant for "database" 🙏

Reply:
Done :)